zed/Dockerfile

```
@@ -0,0 +1,41 @@
+ARG REGISTRY
+ARG RELEASE
+
+FROM ghcr.io/vexxhost/ubuntu-cloud-archive:zed
+# renovate: name=openstack/requirements repo=https://opendev.org/openstack/requirements.git branch=unmaintained/zed
+ARG REQUIREMENTS_GIT_REF=b4513973fdd1c3b43159fe0e02fa73d845a9c1ba
+ADD --keep-git-dir=true https://opendev.org/openstack/requirements.git#${REQUIREMENTS_GIT_REF} /src/requirements
+RUN cp /src/requirements/upper-constraints.txt /upper-constraints.txt
```

I have a feeling the git cloning hasn't been reliable, we can replace this with that:

```suggestion
ADD https://opendev.org/openstack/requirements/raw/commit/${REQUIREMENTS_GIT_REF}/upper-constraints.txt /upper-constraints.txt
```